docs/can-guides/commitment/recipes/cta-bus-map/cta-bus-map.md

```diff
@@ -68,7 +62,7 @@ __A Basic CanJS Setup__
   ```
 
 - CanJS uses [can-stache] to render data in a template
-  and keep it live.  Templates can be authored in `<script>` tags like:
+  and keep it live.  Templates are authored in `<script>` tags like:
```

Comment:
they shouldn't be in a real app.

Reply:
well, they ARE authored in `<script>` tags _for this example_ which is why I found this sentence confusing — perhaps you could add a short blurb to qualify that a real app would organize templates into files.